### PR TITLE
Fixing product-iots#1315

### PR DESCRIPTION
**Pull request by dilee**
Fixed https://github.com/wso2/product-iots/issues/1315

### CODE REVIEW

**Review by harshanL — APPROVED** [review]

File: components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.view/public/css/main.css [hunk 1]
```diff
@@ -142,6 +142,7 @@
 }
 .tab-actions {
 	margin: 0px;
+    margin-bottom: 10px;
```
Comment (sinthuja) [inline components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.view/public/css/main.css:145]: Line indentation formatting issue, please fix this.

File: components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.view/public/js/device-view.js [hunk 2]
```diff
@@ -267,3 +271,47 @@ function loadPolicyCompliance() {
...
+            function (data, textStatus, jqXHR) {
+                if (jqXHR.status == 200 && data) {
+                    data = JSON.parse(data);
+                    $("#apps-spinner").addClass("hidden");
+                    if (data.length > 0) {
+                        for (var i = 0; i < data.length; i++) {
+                            data[i]["name"] = decodeURIComponent(data[i]["name"]);
+                            data[i]["platform"] = deviceType;
+                        }
+
```
Comment (sinthuja) [inline components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.view/public/js/device-view.js:295]: Remove the unnecessary blank line.

File: components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.view/public/css/main.css (hunk 1 above)
Comment (dilee) [reply components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.view/public/css/main.css:145]: Fixed in https://github.com/wso2/carbon-device-mgt/pull/950

File: components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.view/public/js/device-view.js (hunk 2 above)
Comment (dilee) [reply components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.view/public/js/device-view.js:295]: Fixed in https://github.com/wso2/carbon-device-mgt/pull/950

**Pull request merged**